pulpcore/pulpcore/etc/pulp/server.yaml

```diff
@@ -85,6 +85,15 @@
 # server:
 #   working_directory: /var/cache/pulp
 
+# Content Application
+#
+# `content`: The content serving application.
```

Comment:
Django recommends settings names be uppercase[0]
[0] https://docs.djangoproject.com/en/1.11/topics/settings/#creating-your-own-settings

Reply:
I understand but uppercase names in .yml files would be unconventional.  I think, the code that updates the settings.py with the server.yml converts to uppercase.  It's unfortunate that the JWT settings were added as uppercase because it introduces inconsistency.  What do you think about merging as is and adding an issue to standardize this?

Reply:
That sounds good. I'll can file a new issue for this.

Reply:
Thanks, @werwty!